index.js

```diff
@@ -99,6 +99,7 @@ ZongJi.prototype._isChecksumEnabled = function(next) {
         // MySQL < 5.6.2 does not support @@GLOBAL.binlog_checksum
         return next(false);
       }
+      // TODO: Remove throws to make the module safe
```

Comment:
This `throw` and the 2 following (@ line 115 and line 129) could be reported with a callback on the `ZongJi` constructor. This would require adding a third argument to the constructor.